source/agora/flash/Channel.d

```diff
@@ -2206,6 +2211,18 @@ LOuter: while (1)
         return this.channel_updates[$ - 1].settle_tx;
     }
 
+    version (unittest)
+    public Transaction getLastUpdateTx ()
+    {
+        return this.channel_updates[$ - 1].update_tx;
+    }
+
+    version (unittest)
+    public Transaction getTriggerTx ()
+    {
+        return this.channel_updates[0].update_tx;
+    }
```

Comment:
Uuuuuhhhhh we really shouldn't have those

Reply:
Alternative is arbitary sleeps, pick your poison:)
We already had similar helpers anyways.

Reply:
Ok, I might have talked too soon. Good new incoming.